### PR TITLE
Feature/routing child dashboard buttons to correct  paths

### PR DESCRIPTION
**Pull request by artofmayhem**
This PR sets connects the Leaderboard component into the child dashboard and points buttons towards incoming user flow paths /join and /change-avatar. 

This ensures that the child dashboard is now pointing fully into current and intended user flow. 

/change-avatar page is rendering and awaiting an under construction template addition

/join is 404 needing a page created using under construction template

### CODE REVIEW

File: src/components/pages/ChildDashboard/RenderChildDashboard.js [hunk 1]
```diff
@@ -19,15 +19,15 @@ const RenderChildDashboard = props => {
   };
 
   const handleJoinSquad = e => {
-    push('/child/join');
+    push('/join');
```
Comment (just-kato) [inline src/components/pages/ChildDashboard/RenderChildDashboard.js:22]: We talked about removing the child route and you executed that task quickly. Great job as always Tony

File: src/components/pages/Leaderboard/RenderLeaderboard.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const RenderLeaderboard = props => {
   const { push } = useHistory();
 
   const dashboard = () => {
-    push('/child/dashboard');
+    push('/dashboard');
```
Comment (just-kato) [inline src/components/pages/Leaderboard/RenderLeaderboard.js:15]: Nice work

File: src/index.js [hunk 3]
```diff
@@ -288,14 +288,14 @@ function App() {
...
           exact
-          path="/child/leaderboard"
+          path="/leaderboard"
           component={() => (
             <Leaderboard LoadingComponent={ChildLoadingComponent} />
           )}
         />
         {/* INCOMPLETE page exists. NEEDS DATA/ STYLE New path to point to /gameplay/dashboard/change-avatar NOT for DELETE */}
         <ProtectedRoute
-          path="/child/change-avatar"
```
Comment (just-kato) [inline src/index.js:298]: Good job

**Review by just-kato — APPROVED** [review]

**Comment by kendra4227** [comment]
Good job on changing the child endpoints.

**Review by kendra4227 — APPROVED** [review]
Good job on the change of endpoints

**Pull request merged**